servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -2349,6 +2391,54 @@ Policy updateTemplatePolicy(Policy policy, String domainName, String policyName,
         return templatePolicy;
     }
     
+    ServiceIdentity updateTemplateServiceIdentity(ServiceIdentity serviceIdentity, String domainName, String serviceIdentityName, List<TemplateParam> params) {
+        ServiceIdentity templateServcieIdentity = new ServiceIdentity().setName(ZMSUtils.serviceResourceName(domainName, serviceIdentityName));
+        templateServcieIdentity.setDescription("");
```

Comment:
no need to default to empty string.  we should be able just to call:

templateServiceIdentity.setDescription(serviceIdentity.getDescription());